src/utils/api_request_helpers.ts

```diff
@@ -0,0 +1,124 @@
+import { error } from 'console'
+import dotenv from 'dotenv'
+
+// Load environment variables from .env file
+dotenv.config()
+
+//Interfaces
+
+export interface Racemap_event {
+    name: string
+    r_id: string
+}
+
+export interface Event_shadow_coordinates {
+    event_id: string
+    lat: number
+    lng: number
+}
+
+// Function that creates an auth header for requests
+export const generate_authentication_header = async (): Promise<string> => {
+    // Retrieve the environment variable
+    const user_email = process.env.USER_EMAIL
+    const user_password = process.env.USER_PASSWORD
```

Comment:
If no values are set, the values can be set either with 'const user_email = process.env.USER_EMAIL || 'no-email-provided@example.com' or with 'const user_password = process.env.USER_PASSWORD || 'no-password-provided'.
const user_password = process.env.USER_PASSWORD || 'no-password-provided' or with 'const user_email = process.env.USER_EMAIL || null
const user_password = process.env.USER_PASSWORD || null'. Otherwise no auth header can be sent. How is this supposed to work?